openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -607,6 +612,22 @@ def post(self, request):
         mark_user_change_as_expected(user.id)
         return response
 
+    def _handle_country_code_validation(self, request, data):
+        # pylint: disable=no-member
+        errors = {}
+        country = data.get('country')
+        is_authn_mfe = data.get('is_authn_mfe', 'false')
+
+        if is_authn_mfe == 'true':
```

Comment:
Shafqat I am getting rid of `is_authn_mfe` in this PR: https://github.com/openedx/edx-platform/pull/30937
We don't need it anymore.

Reply:
@zainab-amir  why was this used? just asking for knowledge.

Reply:
To filter error messages. We were running old and new messages in parallel during our redesign work

Reply:
@zainab-amir Updated the logic.